packages/lexical-playground/src/plugins/ActionsPlugin.jsx

```diff
@@ -126,16 +126,18 @@ export default function ActionsPlugin({
             'action-button action-button-mic ' +
             (isSpeechToText ? 'active' : '')
           }
-          title="Mic"
-          aria-label="Mic">
+          title="Speect To Text"
```

Comment:
Huge a11y win.

Reply:
Shouldn't this be "Speech"?
```suggestion
          title="Speech To Text"
```